rllib/agents/trainer.py

```diff
@@ -215,8 +215,10 @@
     # Experimental flag to speed up sampling and use "trajectory views" as
     # generic ModelV2 `input_dicts` that can be requested by the model to
     # contain different information on the ongoing episode.
+    # None (default): Automatically determine, whether the API should be used
+    #   or not. This is currently based on the algorithm and framework used.
     # NOTE: Only supported for PyTorch so far.
-    "_use_trajectory_view_api": False,
+    "_use_trajectory_view_api": None,
```

Comment:
Please set this to False by default and remove the None option.

Reply:
All done.